client/cli/src/main.rs

```diff
@@ -76,6 +76,9 @@ enum Command {
 
     /// Wait for the fed to reach a consensus block height
     WaitBlockHeight { height: u64 },
+
+    /// List registered gateways
+    Gateways,
```

Comment:
Should probably be called something like `FederationGateways` since it fetches from the federation and doesn't just list what's already in the DB.

Reply:
The explicitly named `mint-client-cli federation-gateways` is nice but really painful to type and I imagine most folks would end up aliasing it.

Further, I was of a mind to use the same cli command for listing both federation gateways, and the subset that has been sourced and saved to the client DB.
If we consider gateways in the DB as 'active' gateways, from the clients perspective, then I'm proposing we build an experience like `mint-client-cli gateways` and `mint-client-cli gateways -- active`.

Reply:
@elsirion, @justinmoon, what's your take on the idea of gateways vs active gateways from client perspective. Does that categorization make sense?